test/agentchat/contrib/capabilities/test_context_handling.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+import os
+import sys
+from autogen import token_count_utils
+from autogen.agentchat.contrib.capabilities.context_handling import TransformChatHistory
+
+
+def test_transform_chat_history():
```

Comment:
It seems that this test only covers TransformChatHistory. The hook part is not tested. Shall we add a test about that part?